sonar-core/src/main/resources/org/sonar/l10n/core.properties

```diff
@@ -2266,17 +2276,31 @@ metric.generated_ncloc.name.suffix=generated lines of code
 metric.generated_ncloc.description=Generated non Commenting Lines of Code
 metric.generated_ncloc.abbreviation=Gen. LOC
 
+metric.last_commit_date.name=Date of Last Commit
+
```

Comment:
Forget, as the metric is hidden I think no bundle should be added for this metric